PhilipDobranowski/lab2_part1/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.24)
+
+project(hello_world LANGUAGES CXX)
+
+add_library(hello_world_lib SHARED src/hello_world.cpp)
+target_include_directories(hello_world_lib PUBLIC include)
+
+add_executable(test src/main.cpp)
+target_link_libraries(test PUBLIC hello_world_lib)
```

Comment:
Nice job on this, you are so close! You are missing one step in your CMake description. You have successfully built the project, but you are forgetting one step. Hint: currently, your CMake description does not add your built package to the list of all your current downloaded packages. This is the reason why you are getting your error.

Reply:
Why is this now needed in this lab, but wasn't in lab 1? Does this allow Nix to access our library when it's currently only accessible to our binary?

Reply:
> Nice job on this, you are so close! You are missing one step in your CMake description. You have successfully built the project, but you are forgetting one step. Hint: currently, your CMake description does not add your built package to the list of all your current downloaded packages. This is the reason why you are getting your error.

Why is this now needed in this lab, but wasn't in lab 1? Does this allow Nix to access our library when it's currently only accessible to our binary?